<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 7px;
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="slabel">
         <property name="styleSheet">
          <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	color: rgb(70, 140, 0);
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
         </property>
         <property name="text">
          <string>Save Location: C:\</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="browseButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(70, 140, 0);
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(85, 150, 100);
}

QPushButton:pressed{
	background-color: rgb(86, 91, 89);
}
</string>
         </property>
         <property name="text">
          <string>Choose Default Save Location</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>390</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(70, 140, 0);
font: 10 11pt &quot;Segoe UI Semibold&quot;;</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Function</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="10" column="0">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="plotButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(70, 140, 0);
	color: rgb(255, 255, 255);
	font: 12 15pt &quot;Segoe UI Semibold&quot;;
}
QPushButton:hover{
	background-color: rgb(85, 150, 100);
}

QPushButton:pressed{
	background-color: rgb(86, 91, 89);
}
</string>
             </property>
             <property name="text">
              <string>Plot</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="13" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>143</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="13" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>144</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="12" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>202</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
             </property>
             <property name="text">
              <string>X Axis End</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
             </property>
             <property name="text">
              <string>Y Axis End</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
             </property>
             <property name="text">
              <string>Type Funtion to Plot</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QFrame" name="frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame{

	border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	
	color: rgb(255, 255, 255);
	
	font: 12 15pt &quot;Segoe UI Semibold&quot;;
}
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="maximumSize">
                 <size>
                  <width>51</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	color: rgb(70, 140, 0);
	border: none;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
                </property>
                <property name="text">
                 <string>y =</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="functionEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
font: 10 15pt &quot;Segoe UI Semibold&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>x</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
             </property>
             <property name="text">
              <string>X Axis Start</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">    border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border: none;
	padding: 5px;
	font: 12 15pt &quot;Segoe UI Semibold&quot;;

	
	</string>
             </property>
             <property name="text">
              <string>Y Axis Start</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="clearButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(70, 140, 0);
	color: rgb(255, 255, 255);
	font: 12 15pt &quot;Segoe UI Semibold&quot;;
}
QPushButton:hover{
	background-color: rgb(85, 150, 100);
}

QPushButton:pressed{
	background-color: rgb(86, 91, 89);
}
</string>
             </property>
             <property name="text">
              <string>Clear Plot</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QSpinBox" name="x_startSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="x_endSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QSpinBox" name="y_startSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="y_endSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>History</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="historyTextEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>browseButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
